packages/components/ovh-shell/src/message-bus/direct-client.ts

```diff
@@ -26,4 +26,9 @@ export default class DirectClientMessageBus implements IMessageBus {
   onReceive<T>(callback: (message: T) => void): void {
     this.listeners.push(callback);
   }
+
+  cleanup(): void {
+    this.listeners = [];
```

Comment:
May be we could implement something to unregister listener before set array reference ?